model/src/w3servmd.F90

```diff
@@ -10,9 +10,9 @@ MODULE W3SERVMD
 !/                  +-----------------------------------+
 !/
 !/    For update log see individual subroutines.
-!/    12-Jun-2012 : Add /RTD option or rotated grid option. 
+!/    12-Jun-2012 : Add /RTD option or rotated grid option.
```

Comment:
Most changes are white space or style and are to me seem unrelated to the PR.   I'm wondering if it's worth keeping these changes in the PR or if it would be better to revert this file to its original state which would help keep this branch and develop in sync.